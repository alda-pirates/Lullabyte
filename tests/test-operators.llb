void main()
{
	print(3+5);
	print(9-2);
	print(4*6);
	print(10/2);
	print(4%2);
	print(true || true);
	print(true || false);
	print(false && true);
	print(true && true);
	print(3<5);
	print(8<=2);
	print(19>3);
	print(3>=5);
	print(4==4);
	print(1!=2);
	print(true==true);
	print(false==true);
	print(1.2+2.5);
	print(9.0-2.4);
	print(0.5*0.5);
	print(2.5/0.25);
	print(4.4==2.4);
	print(4.000!=2.04);
	print(2.20<4.9);
	print(0.0>1.3);
	print(190.3>=190.3);
	print(100.2<=3.3);
	print(!true);
	print(-4);
	print(-4.4);
	print(2+2.2);
	print(3.5+2);
	print(3.5-2);
	print(1-.5);
	print(6.1*3);
	print(3*4.2);
	print(1/4.0);
	print(8.2/4);
	print(9.0==9);
	print(9==9.1);
	print(1.0!=1);
	print(2!=1.1);
	print(1.1<3);
	print(4<3.2);
	print(4>5.1);
	print(5.1>4);
	print(10.<=10);
	print(11<=4.1);
	print(2.3>=6);
	print(9>=8.2);
}
