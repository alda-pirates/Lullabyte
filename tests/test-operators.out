8
7
24
5
0
true
true
false
true
true
false
true
false
true
true
true
false
3.7
6.6
0.25
10.
false
true
true
false
true
false
false
-4
-4.4
4.2
5.5
1.5
0.5
18.3
12.6
0.25
2.05
true
false
false
true
true
false
false
true
true
false
false
true
